src/classrooms/classrooms.controller.ts

```diff
@@ -0,0 +1,22 @@
+import { Body, Controller, Post, Req, UseGuards } from "@nestjs/common";
+
+import { ITokenizedUser } from "@/auth/auth.interfaces";
+import { JwtAuthGuard } from "@/auth/guards/jwt-auth.guard";
+
+import { CreateClassroomDto } from "./classrooms.dto";
+import { ClassroomsService } from "./classrooms.service";
+
+interface IRequestUser extends Request {
+  user: ITokenizedUser;
+}
+
+@UseGuards(JwtAuthGuard)
+@Controller("classrooms")
+export class ClassroomsController {
+  constructor(private readonly classroomsService: ClassroomsService) {}
+
+  @Post()
+  addClassroom(@Req() req: IRequestUser, @Body() classroomDto: CreateClassroomDto) {
```

Comment:
```suggestion
  createClassroom(@Req() req: IRequestUser, @Body() classroomDto: CreateClassroomDto) {
```

Use @CurrentUser decorator if you need current user id. Can find it in the backend template